app/views/layouts/application.html.erb

```diff
@@ -3,9 +3,21 @@
 <head>
   <title>RutiBookstore</title>
   <%= csrf_meta_tags %>
+  <%= javascript_include_tag "application", "data-turbolinks-track" => true %>
```

Comment:
it looks like you removed `turbolinks` (which you should!), so i'm not sure if this old reference to it here will cause you problems, but you can probably safely remove it

`<%= javascript_include_tag "application" %>`